mk/config.mk

```diff
@@ -62,7 +62,7 @@ SPLINCLUDE    := \
 		-I$(SRCTREE)/include/arch/$(PLATFORM)/ \
 		-I$(SRCTREE)/include/openssl/
 
- COMM_FLAGS := -nostdinc  $(COMPILEINC) \
+ COMM_FLAGS := -ffreestanding $(COMPILEINC) \
```

Comment:
I think you updated the wrong line here. This is the ARM section of the Makefile (which is unused).